app/permissions/models.py

```diff
@@ -0,0 +1,12 @@
+from app import db
+
+from app.lib.models import Base
+
+
+class TodoListPermission(Base):
```

Comment:
`userId` and `todoListId` should be enough to grant sharing permissions, but if you would like to be fancy, we could add like a "role" column that lets you share to someone but grant explicit permissions like "view", "modify", and "admin".